ta/pkcs11/src/processing.h

```diff
@@ -54,4 +54,19 @@ enum pkcs11_rc step_symm_operation(struct pkcs11_session *session,
 
 enum pkcs11_rc tee_init_ctr_operation(struct active_processing *processing,
 				      void *proc_params, size_t params_size);
+
+/*  Digest operations util */
+bool processing_is_tee_digest(uint32_t proc_id);
+
+enum pkcs11_rc init_digest_operation(struct pkcs11_session *session,
+				     enum processing_func function,
+				     struct pkcs11_attribute_head *proc_params,
+				     struct pkcs11_object *obj);
```

Comment:
will digest initialization ever use a (key) object reference?
If not, prefer remove `obj` argument.

Reply:
I don't think so. Just wanted to keep the signature same but can remove.

I'll suppose then I remove the `function` too. And same story for `step_digest_operation()`.

Reply:
'signature', you mean that function prototypes could look like?
i don't think it would add value. but that's just my feeling.

Reply:
function prototype -- I'll clean it up.